### DISCUSSION TITLE
RabbitMQ deployed with certificate name different from server name

### DISCUSSION THREAD

**Discussion by imGannicus** [rabbitmq/rabbitmq-server#12180] [Questions]
Can RabbitMQ deployed with certificate name different from server name like server name rabbitserver.qa.local but third-party certificate name is *.qa.test.rabbit.com (wild card certificate)?
Its ok to add even a DNS entry if required to resolve it.

**Comment by mkuratczyk** [comment]
Yes. For example we have a wildcard cert in this example:
https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/tls

**Comment by michaelklishin** [comment]
@imGannicus you have already asked the same question in https://github.com/rabbitmq/rabbitmq-server/discussions/12164.
